feat: add release note isMarketplaceResponsibleForTaxes (#1041)
vtexdocs/dev-portal-content: docs/release-notes/2024-03-19-new-field-isMarketplaceResponsibleForTaxes-checkout-api.md

---

> title: "Tax Service"

Reviewer: Achei esse título pouco explicativo... Talvez algo tipo "New field for tax calculation on Checkout API" seja melhor

Author: Obrigada pela sugestão! Foi total falta de atenção aqui, eu acabei ajustando a slug, mas esqueci do title

slug: "2024-03-19-new-field-isMarketplaceResponsibleForTaxes-checkout-api"

> type: "improvement"

Reviewer: ```suggestion
type: "added"
```

createdAt: ""
hidden: false

> excerpt: "Introducing a new field, isMarketplaceResponsibleForTaxes, in the Checkout API orderForm. Easily determine tax calculation responsibility between marketplace and seller."

Reviewer: Achei o excerpt longo, recomendo reduzir para uma frase. Testei usando o inspect e ficaria com 3 linhas:

![image](https://github.com/vtexdocs/dev-portal-content/assets/77292838/7b3cc864-4b54-4b9c-8897-0e4f006639a1)

---

> We have added a new field, `isMarketplaceResponsibleForTaxes`, to the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api#overview) `orderForm`. This feature determines whether the marketplace or the seller is responsible for calculating taxes for the products.

Reviewer: ```suggestion
We have added the `isMarketplaceResponsibleForTaxes` property to the [Checkout API](https://developers.vtex.com/docs/api-reference/checkout-api#overview) `orderForm`. This new field is a boolean that indicates whether the marketplace is responsible for calculating taxes for the products (`true`) or if the responsibility lies with the seller (`false`).
```

> For more information on how to use this field, please refer to the [Tax Service Specification](https://developers.vtex.com/docs/guides/tax-services-specification) article.

Reviewer: ```suggestion
For more information on how to use it, please refer to the [Tax Service Specification](https://developers.vtex.com/docs/guides/tax-services-specification) guide.
```

> >ℹ️ This is important for US merchants who must comply with some legislation regarding tax redemption.

Reviewer: - Não adicionaria isso como um callout, mas talvez em um segundo parágrafo para dar mais contexto.
- Queremos falar diretamente que é para US merchants? Pode ser que outros países também precisem disso, não?
- Removeria o "some"
